app/[[...path]]/page.tsx

```diff
@@ -141,9 +140,7 @@ export async function generateMetadata({params}: MetadataProps): Promise<Metadat
       const guideOrPlatform = getCurrentPlatformOrGuide(rootNode, params.path);
       title =
         pageNode.frontmatter.title +
-        (guideOrPlatform
-          ? ` | Sentry for ${formatGuideOrPlatformTitle(guideOrPlatform.name)}`
-          : '');
+        (guideOrPlatform ? ` | Sentry for ${pageNode.frontmatter.title}` : '');
```

Comment:
You need to get that info from `guideOrPlatform`, otherwise it will always take the current page's title

See:
https://sentry-docs-git-cg-refactor-page-titles.sentry.dev/platforms/javascript/install/

<img width="922" alt="Screenshot 2024-09-04 at 15 14 15" src="https://github.com/user-attachments/assets/343fabb5-b173-4ddd-93be-0efb33c835be">

<img width="549" alt="Screenshot 2024-09-04 at 15 14 33" src="https://github.com/user-attachments/assets/07c2d02d-8fbe-4e41-8aef-fb0227262344">

Reply:
🫠